transactions/buyleaseForSaleDapper.cdc

```diff
@@ -0,0 +1,38 @@
+import FindMarket from "../contracts/FindMarket.cdc"
+import FIND from "../contracts/FIND.cdc"
+import Profile from "../contracts/Profile.cdc"
+import DapperUtilityCoin from "../contracts/standard/DapperUtilityCoin.cdc"
+import FindLeaseMarketSale from "../contracts/FindLeaseMarketSale.cdc"
+import FindLeaseMarket from "../contracts/FindLeaseMarket.cdc"
+
+//TODO: test, and rename to Dapper, repeat for other tx
+transaction(dapperAddress: Address, leaseName: String, amount: UFix64) {
```

Comment:
this should be called merchantAddress or?

Reply:
This is not needed when I look again at it. 
I just deleted that